libcxx/test/std/utilities/optional/optional.object/optional.object.ctor/deduct.verify.cpp

```diff
@@ -13,25 +13,26 @@
 // template<class T>
 //   optional(T) -> optional<T>;
 
-#include <optional>
 #include <cassert>
+#include <optional>
 
 struct A {};
 
-int main(int, char**)
-{
-//  Test the explicit deduction guides
+int main(int, char**) {
+  //  Test the explicit deduction guides
 
-//  Test the implicit deduction guides
-    {
-//  optional()
-    std::optional opt;   // expected-error-re {{no viable constructor or deduction guide for deduction of template arguments of '{{(std::)?}}optional'}}
-    }
+  //  Test the implicit deduction guides
 
-    {
-//  optional(nullopt_t)
-    std::optional opt(std::nullopt);   // expected-error-re@optional:* {{static assertion failed{{.*}}instantiation of optional with nullopt_t is ill-formed}}
-    }
+  // clang-format off
+  {
+    //  optional()
+    std::optional opt; // expected-error-re {{no viable constructor or deduction guide for deduction of template arguments of '{{(std::)?}}optional'}}
+  }
 
+  {
+    //  optional(nullopt_t)
+    std::optional opt(std::nullopt); // expected-error-re@optional:* {{static assertion failed{{.*}}instantiation of optional with nullopt_t is ill-formed}}
```

Comment:
Also, given we expect the error message being emitted for `optional:*`, we can move the comment in a new line (in a subsequent PR), like the following.
```C++
    std::optional opt(std::nullopt);
    // expected-error-re@optional:* {{static assertion failed{{.*}}instantiation of optional with nullopt_t is ill-formed}}
```